Gems/Blast/Code/Source/Editor/EditorBlastFamilyComponent.cpp

```diff
@@ -37,9 +37,7 @@ namespace Blast
                     ->Attribute(AZ::Edit::Attributes::Icon, "Icons/Components/Box.png")
                     ->Attribute(AZ::Edit::Attributes::ViewportIcon, "Icons/Components/Viewport/Box.png")
                     ->Attribute(AZ::Edit::Attributes::AppearsInAddComponentMenu, AZ_CRC("Game", 0x232b318c))
-                    ->Attribute(
-                        AZ::Edit::Attributes::HelpPageURL,
-                        "https://o3de.org/docs/user-guide/components/reference/destruction/blast-family/")
+                    ->Attribute(AZ::Edit::Attributes::HelpPageURL, "https://o3de.org/docs/user-guide/components/reference/destruction/blast-family/")
```

Comment:
I think the blast docs got taken down, so this link won't work. Just tried it and it's a 404.